CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.9.0
+ This is a feature release
+* [TK-443](https://tickets.puppetlabs.com/browse/TK-443) Allow disabling autogenerated url based metrics.
```

Comment:
This doesn't fit, I'll lengthen this by a sentence with info about how to disable.